CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+#How to Contribute
+Below are the instructions for making changes to simple chat. There are a few important guidlines we'd wish for you to follow.
+These will make it easier for you to contribute and for us to follow. Thank you for contributing!
+
+##Getting Started
+If you found a bug there are two options. If you have a solution, please follow these steps:
+-Make sure you have a [Github account](https://github.com/join)
+-Fork the repository
+-Edit
+-Create a pull request
```

Comment:
Since line 9 and 10 will be covered in the two sections below, we can replace this with the following information:
Run `npm install && cd frontend && npm install` to install all required packages.